### ISSUE TITLE
Syntax error in shortcode 'media' at line 2, column 133: expecting whitespace!

### ISSUE THREAD

**Issue by bussiere** [getnikola/nikola#2573]
ERROR: Nikola: Shortcode error in file posts/test.rst: 

```
.. title: test
.. slug: test
.. date: 2016-08-21 22:57:58 UTC+02:00
.. tags: titi
.. category:
.. link:
.. description:
.. type: text

{{% pouch %}}
{{% media url="https://gateway.ipfs.io/ipfs/QmPQUodBCkvi3XPaEo41xezVraANufBXKxeC8XGk8T1QmT" %}}
```

I don't understand why there is an error in my file i've copied the media syntax

regards

**Comment by Kwpolska** [comment]
You should avoid shortcodes with reST posts and just use the raw media directive:

    .. media: http://whatever/

But you can’t use quotes around the URL with the shortcode.

**Comment by ralsina** [comment]
This is annoying enough I may bother to implement a shortcode role for reSt that does nothing, like

```:shortcode:`{{% a not broken shortcode %}}```

**Comment by bussiere** [comment]
And so how to give parameter to a shortcode so if i can't use " ?

**Comment by ralsina** [comment]
@bussiere main problem is that if you put a bare URL in reSt it will convert it into a hyperlink, so you end up with something like this:

```{{% media url=<a href="http://...```

So, in reSt, don't use shortcodes if there is an alternative. If this is not reSt, doing it without quotes should work.

**Comment by Kwpolska** [comment]
> This is annoying enough I may bother to implement a shortcode role for reSt that does nothing, like

> `:shortcode:`{{% a not broken shortcode %}}`

That’s funny, because *you already implemented that*, in #2365. You called this `:sc:`.

**Comment by Kwpolska** [comment]
@bussiere This should work:

    {{% media url=https://gateway.ipfs.io/ipfs/QmPQUodBCkvi3XPaEo41xezVraANufBXKxeC8XGk8T1QmT %}}
